### DISCUSSION TITLE
V-for list display not working after Quasar build

### DISCUSSION THREAD

**Discussion by jrhopkins83** [quasarframework/quasar#13208] [CLI - Capacitor / Cordova modes]
I'm seeing a strange behavior between running an app in "dev" vs "build" mode.

I have a component that lists golfers from a vuex getter.  I have no problem when I run the app from a browser or a phone emulator (iPhone or Android) after running Quasar dev.  But when I run the app generated from Quasar build for Android or iOS, the list doesn't display.  (see screenshots below)

In the list header I added the length of the array to verify that the array from the Vuex is actually populated (it is).

As you can see from the emulator screen shot, it appears that Vue is adding a comment line in place of a table row.

I'm using Quasar V2 and Vue V3.

Please help as I'm stuck and can't get my app in the app stores.

Below are the code snippets from the v-for from the parent and the child component displaying the rows, a screenshot from an Android emulator from quasar build, and a screenshot from a chrome browser.

**Parent**
`			<table class="selected-golfers-list" separator bordered dense>
				<golfer
					v-for="(golfer, key) in players"
					:key="key"
					:golfer="golfer"
					:id="key"
					:list="'available'"
					@changePlayerStatus="addPlayer"
					clickable
					v-ripple
					dense
				/>
			</table>`
**Child**
`	<tr
		class="list-golfer"
		clickable
		v-ripple
		dense
		@click="changePlayerStatus(golfer)"
	>
		<td class="name">
			<div>
				{{ golfer.fullName }}
			</div>
		</td>

		<td class="hcp">
			<div class="row" v-if="golfer.fullHandicap">
				<div>Hcp</div>
				<div class="hcp__number">{{ golfer.fullHandicap }}</div>
			</div>
		</td>

		<td>
			<div class="row">
				<q-btn
					flat
					round
					dense
					color="primary"
					@click="changePlayerStatus()"
					:icon="actionIcon"
				/>
			</div>
		</td>
	</tr>
`
**Browser**

<img width="1061" alt="Screen Shot 2022-04-18 at 10 59 55 AM" src="https://user-images.githubusercontent.com/47962891/163852177-e9430102-8217-4f08-b39d-e9d2932d9d1c.png">

**Android emulator**

![Screen Shot 2022-04-18 at 10 45 29 AM](https://user-images.githubusercontent.com/47962891/163852236-ed78d2ec-2092-427c-b460-a8cb187a920b.png)

**Comment by MilosPaunovic** [comment]
It just seems that some of your conditions in `v-if` isn't met, so items aren't rendered into view.
Try building your app with a debug flag to be able to inspect it better (`quasar build --debug`).
